doc/api_2_0_spec/api_2_0_spec.md

```diff
@@ -771,6 +771,33 @@ the returned data would look like:
 }
 ```
 
+## api/v2/stories_public/is_syndicated_ap (POST)
```

Comment:
I'm not sure if `stories_public/is_syndicated_ap` is a good API endpoint name as it doesn't deal with stories as such but rather with a free-form plain text content (which doesn't have to be a story from `stories` table). Personally I see nothing wrong with creating a whole new API "namespace" for helper API calls, e.g. `utils/` or `misc/`. We don't have to squeeze all API calls into the one and only controller just because it's there already, creating new controllers is easy!

Reply:
fixed